htdocs/js/wui.js

```diff
@@ -149,6 +149,13 @@ vz.wui.addEntity = function(entity) {
 		entity.eachChild(function(child) {
 			child.loadTotalConsumption();
 		}, true); // recursive
+	
+	// create push subscriptions
+	entity.subscribe();	
+	entity.eachChild(function(child) {
+		child.subscribe();
+	}, true); // recursive
+	
```

Comment:
Wenn ich den Kanal neu abonniere/erstelle, ist doch immer active=1, oder? Wo sollte ein active=0 herkommen? Cookie gibt es zu diesem Zeitpunkt nicht und was in der DB steht wollen wir ignorieren.